libraries/botframework-connector/src/auth/authenticationError.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IStatusCodeError, StatusCodes } from 'botframework-schema';
+
+export type StatusCode = number;
+
+export class AuthenticationError extends Error implements IStatusCodeError {
+    constructor(
+        public readonly message: string,
```

Comment:
>public readonly [](start = 8, length = 15)

Error should already have a message property. Don't think you need to redefine it here.

Reply:
Ah, that's right. Removed redefinition 👍 git